### ISSUE TITLE
[WAGON-408] ITs for #testSecuredGet() fail with Windows

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-wagon#466]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** opened **[WAGON-408](https://issues.apache.org/jira/browse/WAGON-408?redirect=false)** and commented

```xml
<testcase name="testSecuredGet" classname="org.apache.maven.wagon.providers.http.LightweightHttpWagonWithPreemptiveAuthenticationTest" time="0.148">
    <failure message="not 1 security handler use [] expected:&lt;1&gt; but was:&lt;0&gt;" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: not 1 security handler use [] expected:<1> but was:<0>
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.failNotEquals(Assert.java:283)
	at junit.framework.Assert.assertEquals(Assert.java:64)
	at junit.framework.Assert.assertEquals(Assert.java:195)
	at org.apache.maven.wagon.http.HttpWagonTestCase.testPreemptiveAuthentication(HttpWagonTestCase.java:1545)
	at org.apache.maven.wagon.http.HttpWagonTestCase.testPreemptiveAuthenticationGet(HttpWagonTestCase.java:1532)
	at org.apache.maven.wagon.http.HttpWagonTestCase.runTestSecuredGet(HttpWagonTestCase.java:1066)
	at org.apache.maven.wagon.http.HttpWagonTestCase.testSecuredGet(HttpWagonTestCase.java:1027)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:264)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]></failure>
    <system-out>http://localhost:58257 - Session: Opened  
attempting to create parent directories for destination: wagon-test1432980426073514145txt
Downloading: test-secured-resource from http://localhost:58257

#
Transfer finished. 10 bytes copied in 0.0 seconds
http://localhost:58257 - Session: Disconnecting  
http://localhost:58257 - Session: Disconnected
</system-out>
    <system-err>2014-01-02 15:13:33.461:INFO::jetty-6.1.26
2014-01-02 15:13:33.461:INFO::Started SocketConnector@0.0.0.0:58257
2014-01-02 15:13:33.484:INFO::Stopped SocketConnector@0.0.0.0:58257
</system-err>
  </testcase>
```

---

**Affects:** 2.5, 2.6, 2.10

**Issue Links:**
- [WAGON-474](https://issues.apache.org/jira/browse/WAGON-474) Upgrade and revise all tests for Jetty 8
 (_**"depends upon"**_)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

This issue has been solved by WAGON-474. See [mailing list](http://mail-archives.us.apache.org/mod_mbox/maven-dev/201612.mbox/%3C9d4e5221-d234-3fca-c998-62f9b375751a%40apache.org%3E).

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [5ba0cf33a6f9024c6dbbb3e3e24c978245fa082c](https://git-wip-us.apache.org/repos/asf?p=maven-wagon.git;a=commit;h=5ba0cf33a6f9024c6dbbb3e3e24c978245fa082c).
